Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/GenericConstraintSolver.cpp

```diff
@@ -429,36 +273,17 @@ bool GenericConstraintSolver::solveSystem(const core::ConstraintParams * /*cPara
     current_cp->allVerified = d_allVerified.getValue();
     current_cp->sor = d_sor.getValue();
 
-
-    // Resolution depending on the method selected
-    switch ( d_resolutionMethod.getValue())
+    if (notMuted())
```

Comment:
msg_info() is already something like if(printLog()){}
if you need a different condition you can use msg_info_when(my_condition)